includes/ProductSections/Manager.php

```diff
@@ -0,0 +1,342 @@
+				?>
+                <div class="dokan-form-group">
+                    <label class="dokan-w3 dokan-control-label"><?php esc_html_e( 'Store page product sections', 'dokan-lite' ); ?></label>
+                    <div class="dokan-w5 dokan-text-left">
+				<?php
+            }
+            ?>
+                    <div class="checkbox">
+                        <label>
+                            <input
+                                type="checkbox"
+                                name="product_sections[<?php echo sanitize_key( $section_object->get_section_id() ); ?>]"
+                                value="yes" <?php checked( $show_products_section, 'yes' ); ?>> <?php echo esc_html( $section_object->get_setting_label() ); ?>
+                        </label>
+                    </div>
+            <?php
+        }
+
+        if ( 1 === $setting_count ) {
+			?>
+                    </div>
+                </div>
+			<?php
+        }
+    }
+
+    /**
+     * Save store products section settings input data.
+     *
+     * @since 3.3.7
+     *
+     * @param array $dokan_settings Dokan settings data
+     *
+     * @return array
+     */
+    public function process_store_settings_form_fields( $dokan_settings ) {
+        // Validate nonce
+        if ( ! isset( $_POST['_wpnonce'] ) || ! wp_verify_nonce( sanitize_key( wp_unslash( $_POST['_wpnonce'] ) ), 'dokan_store_settings_nonce' ) ) {
+            return $dokan_settings;
+        }
+
+        foreach ( $this->container as $section_object ) {
+            // check if we've got AbstractProductSection class instance
+            if ( ! $section_object instanceof AbstractProductSection ) {
+                continue;
+            }
+
+            $dokan_settings['product_sections'][ $section_object->get_section_id() ] = 'no';
+
+            if ( ! $section_object->get_show_in_settings() ) {
+                continue;
+            }
+
+            // check if enabled from customizer
+            if ( ! $this->is_enabled( $section_object ) ) {
+                continue;
+            }
+
+            if ( isset( $_POST['product_sections'][ $section_object->get_section_id() ] ) ) {
+                // Add inputs to the store settings args
+                $dokan_settings['product_sections'][ $section_object->get_section_id() ] = 'yes';
+            }
+        }
+
+        return $dokan_settings;
+    }
+
+    /**
+     * Check products block visibility settings by admin and vendor.
+     *
+     * @since 3.3.7
+     *
+     * @param AbstractProductSection $section_object
+     * @param array $store_settings
+     *
+     * @return bool
+     */
+    protected function is_visible( $section_object, $store_settings ) {
+        $customizer_settings = dokan_get_option( 'product_sections', 'dokan_appearance' );
+        // Check if current products section enabled by admin.
+        // for customizer settings default value is hide sections
+        if ( $section_object->get_show_in_customizer() &&
```

Comment:
If the `is_enabled` method gets moved inside the abstract function as mention by the review at `#L304`, we can remove this whole `if` condition by `if ( $section_object->is_enabled() )`. In that case, `$customizer_settings = dokan_get_option( 'product_sections', 'dokan_appearance' );` this line will not be needed.